sdk/nodejs/docker.ts

```diff
@@ -454,9 +451,8 @@ async function dockerBuild(
         buildArgs.push(...["--target", build.target]);
     }
     if (build.cacheFrom) {
-        const cacheFromImages = await cacheFrom;
-        if (cacheFromImages && cacheFromImages.length) {
-            buildArgs.push(...["--cache-from", cacheFromImages.join()]);
+        if (cacheFrom.length) {
+            buildArgs.push(...["--cache-from", cacheFrom.join()]);
         }
```

Comment:
this is where we effectively immediately did the awaiting of `cacheFrom` thus making the concurrency pointless.  It's not like there was other expensive work we were doing in the meantime.  We were simply building up the args to then just pass to `docker build`.  Since that build had a dep on this, it's not like we had any benefit here.  (i.e. it's not like we could kick off the build concurrently while this was executing).